### DISCUSSION TITLE
Request for Assistance in Recovering Access to My GitHub Account

### DISCUSSION THREAD

**Discussion by shenlongviact** [community/community#146324] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Dear GitHub Support Team,

I hope this email finds you well.

I am writing to seek your assistance in recovering access to my GitHub account. Unfortunately, I recently changed my phone and no longer have access to the 2FA application I was using to generate authentication codes for logging into my account. Additionally, I no longer have my recovery codes saved.

I attempted to retrieve my recovery codes through the email sent by GitHub. However, I was prompted to log in to download the codes, which I am unable to do due to the 2FA issue.

I still have full access to the email address associated with my GitHub account and can provide any necessary verification details to confirm my ownership of the account. Additionally, I am currently subscribed to Copilot at $10 per month using my Visa card. As this account is critical to my work, creating a new account is not an option for me.

I would greatly appreciate it if you could guide me through the recovery process or provide an alternative way to regain access to my account. Please let me know if you require any further information from my side.

Thank you very much for your time and support. I look forward to your response.

Best regards,
My github email: [buithanhlong091295@gmail.com](mailto:buithanhlong091295@gmail.com)

**Comment by davevad93** [comment, marked as answer]
Hi @shenlongviact ,

For account related issues it's recommended to contact support by opening a ticket [here](https://support.github.com/contact/account), select "`Two-factor authentication recovery`" and explain your problem.

**Reply by queenofcorgis** [reply]
Hi @shenlongviact,

Thank you for your patience. The community is unable to assist with nor escalate your account-specific issues and ticket . Once opened, it will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
